### PR TITLE
chore: automate npm publishing

### PR DESCRIPTION
**Pull request by erezrokah**
Adds `npm` publishing automation

### CODE REVIEW

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -18,3 +18,14 @@ jobs:
           token: ${{ steps.get-token.outputs.token }}
           release-type: node
           package-name: '@netlify/open-api'
+      - uses: actions/checkout@v2
+        if: ${{ steps.release.outputs.release_created }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```
Comment (erezrokah) [inline .github/workflows/release-please.yml:25]: Uses same Node.js version as the test workflow

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -17,13 +17,13 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - name: Install Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
```
Comment (erezrokah) [inline .github/workflows/test.yml:20]: `v2` is faster

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -17,13 +17,13 @@ jobs:
...
         with:
           node-version: ${{ matrix.node_version }}
       - name: Git checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
```
Comment (erezrokah) [inline .github/workflows/test.yml:24]: `v2` has [improved performance](https://github.com/actions/checkout#checkout-v2)

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -17,13 +17,13 @@ jobs:
...
       - name: Install dependencies
-        run: npm install
+        run: npm ci
```
Comment (erezrokah) [inline .github/workflows/test.yml:26]: `npm ci` for reproducible builds

**Review by JGAntunes — APPROVED** [review]

**Pull request merged**